include/avif/internal.h

```diff
@@ -357,7 +357,8 @@ typedef avifBool (*avifCodecGetNextImageFunc)(struct avifCodec * codec,
 // avifCodecEncodeImageFunc is responsible for automatic tiling if encoder->autoTiling is set to
 // AVIF_TRUE. The actual tiling values are passed to avifCodecEncodeImageFunc as parameters.
 // Similarly, avifCodecEncodeImageFunc should use the quantizer parameter instead of
-// encoder->quality and encoder->qualityAlpha.
+// encoder->quality and encoder->qualityAlpha. If disableLaggedOutput is AVIF_TRUE, then the encoder will emit the output frame
+// without any lag (if supported).
```

Comment:
Document that `disableLaggedOutput` is only used by the first `avifCodecEncodeImageFunc` call (which initializes the AV1 or AV2 encoder) and is ignored by subsequent `avifCodecEncodeImageFunc` calls.

Reply:
Done.